common/containers/Tabs/TransactionStatus/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import TabSection from 'containers/TabSection';
+import {
+  signLocalTransactionSucceeded,
+  TSignLocalTransactionSucceeded,
+  signTransactionFailed,
+  TSignTransactionFailed
+} from 'actions/transaction';
+import { computeIndexingHash } from 'libs/transaction';
+import translate from 'translations';
+import SimpleButton from 'components/ui/SimpleButton';
+import EthTx from 'ethereumjs-tx';
+import classnames from 'classnames';
+import { toBuffer } from 'ethereumjs-util';
+import { getSerializedTransaction } from 'selectors/transaction';
+import { AppState } from 'reducers';
+import './index.scss';
+
+interface StateProps {
+  stateTransaction: AppState['transaction']['sign']['local']['signedTransaction'];
+}
+interface DispatchProps {
+  signLocalTransactionSucceeded: TSignLocalTransactionSucceeded;
+  signTransactionFailed: TSignTransactionFailed;
```

Comment:
We don't want to do anything related to signing on this page, so these aren't the actions you'll want. I don't even think you'll need any actions or redux state here, this can by and large be handled internally in the component.